### ISSUE TITLE
I cannot see some extensions that I can see on the other ntb (with the same Ubuntu/Gnome)

### ISSUE THREAD

**Issue by hawran** [mjakeman/extension-manager#226]
Hi,
I've upgraded two notebooks to last version of Ubuntu and installed extension-manager into both.

To my big surprise I cannot see the same extensions within both notebooks, [workspace-indicator](https://extensions.gnome.org/extension/21/workspace-indicator/) in particular.

The notebook 1:
```
Ubuntu 22.04 LTS
GNOME Shell 42.2
Extension Manager 0.3.0
/org/gnome/shell/disable-extension-version-validation false
/org/gnome/shell/disable-user-extensions false
```

The notebook 2:
```
Ubuntu 22.04 LTS
GNOME Shell 42.2
Extension Manager 0.3.0
/org/gnome/shell/disable-extension-version-validation false
/org/gnome/shell/disable-user-extensions false
```

What am I missing?

**Comment by mjakeman** [comment]
Hi, to clarify is this an issue with the extension not showing up in search results?

The search functionality works independently of your shell version, so in theory they should be identical. It could be an issue with only 10 results being shown, which will be fixed by #221.

**Comment by hawran** [comment]
> Hi, to clarify is this an issue with the extension not showing up in search results?

Hi, yes, I'm talking about search results. (The `Browse` button / the `Search for extensions`... panel)

**Comment by mjakeman** [comment]
Hi @hawran does this still happen with version 0.4? I've reworked search so it may have been fixed in the process

**Comment by hawran** [comment]
> Hi @hawran does this still happen with version 0.4? I've reworked search so it may have been fixed in the process

Hi, I'm on 0.3.0 version, the system still does not offer the newer one...

**Comment by mjakeman** [comment]
Can you see if this is reproducible with the flatpak version?

Since there are so many different factors that can break distro packages, I only take flatpak bug reports since it helps me narrow down whether it's an application problem or not. Otherwise, I would recommend reporting it to the relevant packager (e.g. the Ubuntu issue tracker, AUR, etc)

**Comment by hawran** [comment]
> Can you see if this is reproducible with the flatpak version?
> ...

Well, I'm not going to change the "source" of this package. It seems each package needs its own package system nowadays - flatpak, snap, ... When is this going to end?

However, point taken.

**Comment by mjakeman** [comment]
We did a lot of fixes to search in 0.4.3

These sorts of issues should all be fixed now (fingers crossed)
